testoption.go

```diff
@@ -27,6 +27,18 @@ func StartTimeAt(st time.Time) TestOption {
 	})
 }
 
+// WithMessageComparator returns a test options that configures the comparator
+// to use when comparing messages for equality.
```

Comment:
Check this for readability? I'm not sure, but was this the intended wording?
```suggestion
// WithMessageComparator returns a test option that configures the comparator
// for use when comparing messages for equality.
```

Reply:
perhaps:

```suggestion
// WithMessageComparator returns a test option that sets the comparator
// to be used when comparing messages for equality.
```

Reply:
Approved with your new suggested change.